### DISCUSSION TITLE
How to sort dictionary items alphabetically by key?

### DISCUSSION THREAD

**Discussion by seerwill** [typst/typst#1490] [Q&A] [answered]
I want to sort the output from `dictionary.pairs()` alphabetically, so that I can display it in a table.

 I would expect to be able to do it like this:

```rust
let sorted = my_dictionary.pairs().sorted(k => k.at(0))
```

But I get: 

```
error: cannot order array and array
```

Is there a way to do this?

**Comment by magistau** [comment, marked as answer]
```rs
let sorted = my_dictionary.pairs().sorted(key: k => k.at(0))
```

**Reply by seerwill** [reply]
Interesting, I wouldn't have picked that syntax. Thanks!
